packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -106,6 +106,11 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
       restoreScrollOffset();
   }
 
+  /// If applicable, the velocity at which the scroll offset is currently
+  /// independently changing (i.e. without external stimuli such as a dragging gestures)
+  /// in logical pixels per second for the associated activity
```

Comment:
Documentation must end with punctuation.

The docs should also say what happens if it's not applicable.
Also, what it means for it to be applicable.

Reply:
Also we should say how to get notified when this changes.

Reply:
@Hixie I took a lot of the documentation from the scroll activity velocity. I am wondering if (in addition to how to get notified of changes) the documentation for the scroll position velocity should be simplified to
`Represents the currently operative [ScrollActivity] velocity. If there is no current activity the velocity will be null.`.
If users want more details on velocity they could go look into ScrollActivity. That way we keep any mention of "applicable" or other specific vocab in the class to introduced it. Thoughts?